src/domain/buildingBlocks/Pluto.ts

```diff
@@ -86,26 +76,11 @@ export interface Pluto {
    */
   storeCredential(credential: Credential): Promise<void>;
 
-  /**
-   * Retrieve all stored PRISM DIDs.
-   */
-  getAllPrismDIDs(): Promise<PrismDIDInfo[]>;
```

Comment:
raises a question of if we should have general requirements, if not used by us?
ie should Pluto accomodate developers, or only define what is needed internally?